Day1/Exercise3/solution.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import "fmt"
+
+type Employee interface {
+	Fulltime()
+	Contractor()
+	Freelancer()
+}
+
+type details struct {
+	days,dailywage,workinghour int
+}
+
+func (t details) Fulltime() {
+	fmt.Println(t.days*t.dailywage*t.workinghour)
+}
+
+func (t details) Contractor() {
+	fmt.Println(t.days*t.dailywage*t.workinghour)
+}
+
+func (t details) Freelancer() {
+	fmt.Println(t.days*t.dailywage*t.workinghour)
+}
+func main() {
+
+	var employee string
```

Comment:
THis is not the expected solution for the given problem, you need to have different implementation of Contractor, Freelancer and FullTime of Employee Interface, not creating different memember methods for details